packages/moments/README.md

```diff
@@ -2,37 +2,60 @@
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
 
-@poap-xyz/moments is a [short description of the purpose of the package].
+@poap-xyz/moments is a package to interact with POAP Moments.
 
 ## Features
 
-- Feature 1
-- Feature 2
-- Feature 3
+- Create a Moment attached to a Drop or an specific POAP
+- Fetch multiple Moments
+- Fetch a single Moment
 
 ## Installation
 
+### NPM
+
 ```bash
-npm install @poap-xyz/moments
+npm install @poap-xyz/moments @poap-xyz/utils @poap-xyz/providers axios form-data
 ```
 
-## Usage
+### Yarn
 
-\```python
-from package_name import module
+```bash
+yarn add @poap-xyz/moments @poap-xyz/utils @poap-xyz/providers axios form-data
```

Comment:
Wouldn't `utils` and `providers` packages be installed automatically as dependencies? Moments would not work without them.